### PR TITLE
CNNETANTWO-52 and All subtasks

### PR DESCRIPTION
**Pull request by zoitec**
CNNETANTWO-53 - content list page
CNNETANTWO-54 - content list module
CNNETANTWO-55 - content module unit tests

Pull this branch down and go to http://localhost:5050/list

image

### CODE REVIEW

File: src/modules/article-items/__tests__/article-item.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```
Comment (dsover) [inline src/modules/article-items/__tests__/article-item.spec.js]: please rename the test file and re-do the snapshot to reflect article-items

File: src/redux/reducers/articlesReducer.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+    case types.FETCH_ARTICLES_SUCCESS: {
+        return {
+            ...state,
+            fetching: false,
+            fetched: true,
+            data: action.payload.data
+        }
+    }
+    }
```
Comment (jamsyoung) [inline src/redux/reducers/articlesReducer.js]: The double closing braces at the same indention level look odd.  Typically case blocks are not wrapped in braces.  Are they required here?

File: src/redux/reducers/factorsReducer.js [hunk 3]
```diff
@@ -22,7 +22,6 @@ export default function reducer(state = {}, action) {
             factors: action.payload
         }
     }
-
     }
```
Comment (jamsyoung) [inline src/redux/reducers/factorsReducer.js]: I suspect this is another switch statement with the cases blocked with braces.  Is that required?

**Comment by jamsyoung** [comment]
Love the description.  As we move from API work to front end work, please continue to provide screenshots in the description of the PR showing the changes.

**Pull request merged**